deployments/base/weth/deploy.ts

```diff
@@ -0,0 +1,57 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet, exp } from '../../../src/deploy';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  return deployed;
+}
+
+async function deployContracts(
+  deploymentManager: DeploymentManager,
+  deploySpec: DeploySpec
+): Promise<Deployed> {
+  // Deploy constant price feed for WETH
+  const wethConstantPriceFeed = await deploymentManager.deploy(
```

Comment:
The implication of using a constant price feed is that there will always be a 1:1 exchange rate between WETH and ETH, right?

Reply:
Correct. Base's WETH is a [precompile](https://basescan.org/address/0x4200000000000000000000000000000000000006#code) that implements WETH9, so this invariant should always be true, unless there's some 0-day vulnerability in the WETH9 contract that no one has figured out yet.